uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -529,6 +554,12 @@ define([
           dateObj.UI.isAM = isDayType('half_day_am', leaveRequest, dateObj.date);
           dateObj.UI.isPM = isDayType('half_day_pm', leaveRequest, dateObj.date);
         }
+
+        monthObject = getMonthObjectByDate(moment(dateObj.date), indexedByMonth);
+        if(monthObject) {
```

Comment:
can it be that there is no month object? if that can happen, is it a problem? what does it mean?